sdk/typescript/src/signers/signer.ts

```diff
@@ -11,6 +11,7 @@ import { Base64DataBuffer } from '../serialization/base64';
  * Pair of signature and corresponding public key
  */
 export type SignaturePubkeyPair = {
+  flag: Base64DataBuffer;
```

Comment:
@punwai Sorry for the late review, can we change the type from `Base64DataBuffer` to `enum`? That makes the API more intuitive to use and less error prone.

Reply:
For sure, I will do another PR to fix this.